rasa/utils/tensorflow/model_data.py

```diff
@@ -158,6 +158,13 @@ def __setstate__(self, state: Any, **kwargs: Any) -> None:
         self.units = state[-1]
         super(FeatureArray, self).__setstate__(state[0:-3], **kwargs)
 
+    def __str__(self) -> Text:
```

Comment:
Curious and hoping to learn: Why not `__repr__` instead of `__str__`?

Reply:
Because __repr__ should be unambigous but I'm not printing the actual values here.